src/game/Commands/CharacterCommands.cpp

```diff
@@ -1255,12 +1255,12 @@ bool ChatHandler::GetDeletedCharacterInfoList(DeletedInfoList& foundList, bool u
                 std::copy(list.begin(), list.end(), std::ostream_iterator<int>(accountStream, ","));
                 std::string accounts = accountStream.str();
                 accounts.pop_back();
-                resultChar = CharacterDatabase.PQuery("SELECT guid, deleteInfos_Name, deleteInfos_Account, deleteDate FROM characters WHERE deleteDate IS NOT NULL AND deleteInfos_Account IN (%s) LIMIT 0,50", accounts.c_str());
+                resultChar = CharacterDatabase.PQuery("SELECT `guid`, `deleteInfos_Name`, `deleteInfos_Account`, `deleteDate` FROM `characters` WHERE `deleteDate` IS NOT NULL AND `deleteInfos_Account` IN '%s' LIMIT 0,50", accounts.c_str());
```

Comment:
what are you doing here, why did u remove the brackets around %s, this is where the list will go

Reply:
fixed with https://github.com/vmangos/core/pull/896/commits/c552f3bdd708ebbee45b02b1a864d228c4f6dde0